packages/vitepress-plugin-git-changelog/src/client/components/Contributors.vue

```diff
@@ -20,48 +20,66 @@ const options = inject(InjectionKey, {})
 const { t } = useI18n()
 const rawPath = useRawPath()
 const commits = useCommits(Changelog.commits, rawPath)
+const findRegisteredCreatorByName = (author_name: string) => options.mapContributors?.find(item => item.nameAliases && Array.isArray(item.nameAliases) && item.nameAliases.includes(author_name))
+const findRegisteredCreatorByEmail = (author_email: string) => options.mapContributors?.find(item => item.emailAliases && Array.isArray(item.emailAliases) && item.emailAliases.includes(author_email))
 
-const contributors = computed<ContributorInfo[]>(() => {
-  const map: Record<string, ContributorInfo> = {}
-  commits.value.forEach((c) => {
-    const targetCreatorByName = options.mapContributors?.find(item => item.nameAliases && Array.isArray(item.nameAliases) && item.nameAliases.includes(c.author_name))
-    const targetCreatorByEmail = options.mapContributors?.find(item => item.emailAliases && Array.isArray(item.emailAliases) && item.emailAliases.includes(c.author_email))
+const multipleAuthorsRegex = /^ *?Co-authored-by:(.*)(?:<|\(|\[|\{)(.*)(?:>|\)|\]|\}) *?$/mi
```

Comment:
- Should we add comments to explain what this Regexp does?
- What will happen on Safari when `?=`, `?<=`, `?!`, `?<!` is uses?
- Is `co-authored-by` an acceptable case too?

Reply:
- I added some comments
- The syntax you listed is not in this expression.
- The `i` at the end of the regular expression means case insensitive.